db/migrate/20150911195321_add_budget_roles.rb

```diff
@@ -0,0 +1,24 @@
+class AddBudgetRoles < ActiveRecord::Migration
+  def change
+    User.with_email_role_slug!(
```

Comment:
Because "I expect that a migration will only ever run once in any given environment," I also expect that the environment variables are really only relevant in production, where they can be unset after the migration has run.